db/seeds/dev/moviesSeed.js

```diff
@@ -0,0 +1,28 @@
+const movieData = require('../../data/movies');
+const category = 'family'
+
+const createMovie = async (knex, movie) => {
+  const movieId = await knex('movies').insert({
+    title: movie.title.title,
+    content_rating: movie.certificates && movie.certificates.US[0].certificate,
+    movie_plot: movie.plotSummary && movie.plotSummary.text,
+    brief_description: movie.plotOutline.text,
+    image_poster: movie.title.image.url,
+    imdb_rating: movie.ratings.rating,
+    release_date: movie.releaseDate,
+    runtime: movie.title.runningTimeInMinutes,
+    where_to_watch: "not sure",
+  }, 'id');
```

Comment:
could you expand on what line 15 is doing?

Reply:
I was wondering about this too, I took it from the knex express lesson boilerplate. I just tested removing it and it seems like it changed nothing. Going to remove it from both seed files I have used so far and push back. No sense in having code that doesn't do anything.

Reply:
@BlakeDonn I wondered the same thing when I saw it in the lesson.  I figured having the `id` in there isn't necessary for the scope of our project.